libs/zsdl/build.zig

```diff
@@ -66,25 +70,27 @@ pub const Package = struct {
                 }
             },
             .macos => {
-                exe.addRPath(.{ .path = "@executable_path/Frameworks" });
+                exe.root_module.addRPath(.{ .path = "@executable_path/Frameworks" });
```

Comment:
```suggestion
               exe.root_module.addRPathSpecial("@executable_path/Frameworks");
```
This fixes a compile error on macOS